index.html

```diff
@@ -0,0 +1,166 @@
+                        <h4>Quran Memorization Course</h4>
+                        <p class="intro">This course is designed to attain the Itqan level in Hifz Quran online through one-on-one Classes. The students (adults only) can learn advanced topics in Qiraat with objective to attain perfection of Quran memorization (Itqan).</p>
+                    </li>
+
+                    <li class="pos-rel">
+                        <img src="./assets/mobile/al-noor.png" alt="" width="30px" height="30px">
+                        <h4>Islamic Studies Course</h4>
+                        <p class="intro">You can enjoy and Learn Islam Online with our Islamic Studies program that covers a wide range of subjects that are essential for every Muslim.</p>
+                    </li>
+
+                    <li class="pos-rel">
+                        <img src="./assets/mobile/al-noor.png" alt="" width="30px" height="30px">
+                        <h4>Arabic Language Courses</h4>
+                        <p class="intro">“Indeed, We have made it an Arabic Qur’an that you might understand“. (Al-Zukhruf 43:3)
+
+                        </p>
+                    </li>
+                    </ul>
+                    <button class="btn-second" type="button">START YOUR JOURNEY</button>
+                </div>
+            </div>
+            </div>
+        </section>
+
+        <section id="list">
+            <div class="container-ma">
+                <div class="container-start cont-teach">
+                    <div class="teachers-list">
+                        <h2 class="teach">Teachers Who Inspire</h2>
+                        <hr class="hr-teach">
+                <p class="muslim">Building a strong Muslim community.</p>
+                    <ul class="flex">
+                        
+                    </ul>
+
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="container-main">
+                <div class="container-start">
+                    <h2 class="partners">Partners</h2>
+                <ul class="flex1">
+                    <li>Ornaganization of Islamic Coorperation</li>
+                    <li>Muslim Charity</li>
+                    <li>Islamic Relief Worldwide</li>
+                    <li>Muslim Aid</li>
+                    <li>Muslim World League</li>     
+                </ul>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="container-main footer">
+            <div class="container-start">
+                <label class="logo2" for="logo"><img src="./assets/mobile/logo2.png" alt="logo" width="80px" height="80px"></label>
+                <p class="footer-text">
+                    © 2022 Al-noor Mu'alim al-Qur'an<br>
+Islamic Information Center
+                </p>
+            </div>
+        </div>
+    </footer>
```

Comment:
- [ ] [optional] The footer section on the main page is supposed to have a white background.

Reply:
I will not change that 😠